pallets/asset/src/benchmarking.rs

```diff
@@ -269,20 +243,20 @@ benchmarks! {
         let owner = UserBuilder::default().generate_did().build("owner");
         let ticker = make_asset::<T>(&owner);
 
-    }: _(owner.origin, ticker.clone(), 1_000_000.into())
+    }: _(owner.origin, ticker.clone(), (1_000_000 * POLY).into())
     verify {
         let token = Module::<T>::token_details(ticker);
-        assert_eq!( token.total_supply, 2_000_000.into());
+        assert_eq!( token.total_supply, (2_000_000 * POLY).into());
```

Comment:
Some extra space within parentheses.
```suggestion
        assert_eq!(token.total_supply, (2_000_000 * POLY).into());
```